battleship-client/src/main/java/com/epam/solid/nie/client/ui/Board.java

```diff
@@ -11,20 +11,19 @@
 
 import java.util.ArrayList;
 import java.util.List;
-import java.util.logging.Level;
-import java.util.logging.Logger;
+
+import static java.lang.System.out;
 
 
 /**
  *
  */
 class Board extends Parent {
-    private Logger logger = Logger.getLogger(Board.class.getName());
     private static final int MAX_HEIGHT = 10;
     private static final int MAX_WIDTH = 10;
     private VBox rows = new VBox();
     private boolean enemy;
-    private List<Ship> allShips = new ArrayList<>();
+    List<Ship> allShips = new ArrayList<>();
```

Comment:
I think replacing private by package-private usually does not bode well, though I do not know what impact it has on your code